### DISCUSSION TITLE
Choose version to upgrade to Home Assistant Container LXC update script

### DISCUSSION THREAD

**Discussion by SilverHazeLazers** [tteck/Proxmox#1010] [Ideas]
It would be great if when using the update script for the Home Assistant Container LXC if there user could choose which version they want to upgrade to. I personally try to avoid the first major release so it would be great if I could choose the latest patch version from the previous release

**Comment by tteck** [comment]
The current process of updating a container utilizes the "runlike" command to produce the command line required to run a similar container with the updated image. This means that the docker run command line arguments are derived from the running containers and it is not possible to modify the tags selectively.

You can manually setup a container using whatever tag you choose by
```
docker pull homeassistant/home-assistant:<TAG>
docker rm --force homeassistant
docker run -d \
  --name homeassistant \
  --privileged \
  --restart unless-stopped \
  -v /var/run/docker.sock:/var/run/docker.sock \
  -v /dev:/dev \
  -v hass_config:/config \
  -v /etc/localtime:/etc/localtime:ro \
  -v /etc/timezone:/etc/timezone:ro \
  --net=host \
  homeassistant/home-assistant:<TAG>
```
Example:
To run the beta branch, replace `<TAG>` with "beta". The next time you update, "runlike" will reverse-engineer the docker run command line arguments based on running containers and update the container to the "beta" tag.

If you want to use a specific version, replace `<TAG>` with the desired version, such as "2022.12.9". The container will remain on that version until you manually change it.

**Reply by SilverHazeLazers** [reply]
Perfect, thank you very much @tteck for the assistance and all your work in this repo
